http/preview.go

```diff
@@ -151,6 +155,10 @@ func createPreview(imgSvc ImgService, fileCache FileCache,
 	return buf.Bytes(), nil
 }
 
-func previewCacheKey(fPath string, fTime int64, previewSize PreviewSize) string {
-	return fmt.Sprintf("%x%x%x", fPath, fTime, previewSize)
+func previewCacheKey(file *files.FileInfo, previewSize PreviewSize) (string, error) {	
+	fPath, err := file.Fs.(*afero.BasePathFs).RealPath(file.Path)
```

Comment:
This cast will panic if file.Fs is not an instance of `afero.BasePathFs`. I would define a `RealPath` method on `files.FileInfo` which returns a `BasePathFs.RealPath` if fs is an instance of `BasePathFs` or file.Path as a fallback

Reply:
> This cast will panic if file.Fs is not an instance of `afero.BasePathFs`. I would define a `RealPath` method on `files.FileInfo` which returns a `BasePathFs.RealPath` if fs is an instance of `BasePathFs` or file.Path as a fallback

o1egl, would you please make a PR for this? It's beyond my ability in golang.

Reply:
Personally, I will use this one:
func previewCacheKey(file *files.FileInfo, previewSize PreviewSize) string {	
	return fmt.Sprintf("%x%x%x", filepath.Base(file.Path), file.ModTime.Unix(), previewSize, file.Size)
I think filename & modifed time & file size is enough to tell different images. and the good thing of using this is: when we move or rename a folder with tons of images, it will not regenerate the cache files.